node-scripts/spec/pipeline1/resources/default/.ocd3.yaml

```diff
@@ -0,0 +1,7 @@
+version: 1
+
+build:
+  bash_commands: "mvn clean install -B"
+deploy:
+  bash_commands: "./scripts/deploy.sh"
```

Comment:
Just for the sake of the test resource to be real-world like, shouldn't this be
```yml
bash_commands: "sh ./scripts/deploy.sh"
```
?

Reply:
pointing to the script is usually enough to run the script since we're already in a bash shell